app/views/authentications/new.html.erb

```diff
@@ -52,10 +73,10 @@
       |,
       data: { turbo_submits_with: "Authenticating..." }
     %>
-  <% end %>
-  <% if Feature.password_reset_email? %>
-    <span class="text-sm text-center">
-      <%= link_to "Forgot your password?", new_password_reset_path, class: "underline" %>
-    </span>
-  <% end %>
+<% end %>
+<% if Feature.password_reset_email? %>
+<span class="text-sm text-center">
+  <%= link_to "Forgot your password?", new_password_reset_path, class: "underline" %>
+</span>
```

Comment:
Did you mean to remove this layer of indentation inside the conditionals? My instinct would be to keep it, but was it causing some issues?